CodeStarterCamp_Week1/main.swift

```diff
@@ -1,30 +1,5 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week1
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
 
 import Foundation
 
-func drawIce(height:Int){
-    for _ in 1...height{
-        for _ in 1...11{
-            print("*",terminator: "")
-        }
-        print()
-    }
-}
-func drawStick(height:Int){
-    for _ in 1...height{
-        print("    | |")
-    }
-}
 
-func drawIceCream(iceHeight : Int, stickHeight : Int){
-    drawIce(height : iceHeight)
-    drawStick(height: stickHeight)
-}
-
-drawIceCream(iceHeight: 8, stickHeight: 4)
+pepero(12,"***","#",4)
```

Comment:
위에 코멘트 대로 코드를 수정하셨다면, main.swift 에서는
`4가지 예시` 모두 출력해보면 더 좋지 않을까요~? 😄 
이왕 메서드 만들었으니, 다양한 쇼케이스를 보여주는 거죠.

```suggestion
drawPepero(bodyLength: 10, body: "***", stickLength: 4) // 일반 빼빼로
drawPepero(bodyLength: 12, body: "***", topping: "&", stickLength: 4) // & 토핑 빼빼로
drawPepero(bodyLength: 12, body: "***", topping: "#", stickLength: 6) // # 토핑 빼빼로
drawPepero(bodyLength: 6, body: "|0|", stickLength: 4) // 누드 빼빼로
```